.gitignore

```diff
@@ -27,4 +27,5 @@ node_modules
 # Users Environment Variables
 .lock-wscript
 
+.idea
```

Comment:
Please don't put your personal IDE into .gitignore; you should [use a global configuration](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) instead.
Ideally .gitignore only contains things common to everybody using the repo.

Reply:
Cool, thanks, will remove!